strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-nuget-layout-provider/src/test/java/org/carlspring/strongbox/artifact/generator/NugetArtifactGenerator.java

```diff
@@ -54,6 +56,8 @@
 
     private static final String PACKAGING_NUPKG = "nupkg";
 
+    private static final int DEFAULT_SIZE = 1000000;
```

Comment:
Similarly to the previous pull request, could you please change this:
```
    private static final int DEFAULT_SIZE = 1000000;
```
 to:
```
    private static final int DEFAULT_BYTE_SIZE = 1000000;
```
?

Thanks! :)